python/examples/dipole_in_vacuum_cyl_axisymmetric.py

```diff
@@ -0,0 +1,288 @@
+"""Radiation pattern of an axisymmetric dipole in cylindrical coordinates.
```

Comment:
Change axisymmetric to "on-axis" in the filename, within the script, and in plot titles.

Similarly for "off-axis"

Reply:
Fixed.